### PR TITLE
Task-38697: In the TASKS tab, the task marked as completed didn't dis…

### PR DESCRIPTION
**Pull request by lbengayed**
…appear directly
This PR fix the display of task from the list when it marked as completed

### CODE REVIEW

File: task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue [hunk 1]
```diff
@@ -17,6 +18,11 @@
         default: () => [],
       }
     },
+    methods:{
+      updateTaskCompleted(e){
+        window.setTimeout(() => this.tasks = this.tasks.filter((t) => t.task.id !== e.id), 500);
```
Comment (ahamdi) [inline task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue]: Should this take in con,sideration wether the show completed filter is activated or not ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue (hunk 1 above)
Comment (lbengayed) [reply task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue]: it shouldn't take considered when the show completed filter is activated, I ass PR for this, thanks

File: task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue [hunk 2]
```diff
@@ -17,6 +18,27 @@
...
+      },
+      filterCompletedActive() {
+        this.tasks.forEach(task_elm => {
+          while (task_elm.completed === true) {
+            this.filterTaskCompletedActive = true;
+            break;
+          }
+        })
+      },
+    }
```
Comment (ahamdi) [inline task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue:41]: Combine both functions and use the filter to remove all completed tasks based on their property

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue (hunk 2 above)
Comment (lbengayed) [reply task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue:41]: when the filtre marked as completed active, we shouldn't  remove the task from the list when we clique in the button marked as completed, for that I add  this method filterCompletedActive to test if the filtre active or not

**Pull request closed without merging**